webapp/app/js/models/resultsets.js

```diff
@@ -59,32 +58,56 @@ treeherder.factory('ThResultSetModel', [
         'exclusion_state'
     ];
 
-    var registerResultSetPollers = function(){
+    var registerResultSetPollers = function() {
```

Comment:
Ed: I was wrong, the resultsets are not loaded before we first call this function.  This function performs the first load of resultsets on page load.  So I can't check ``doResultSetPolling`` here.   It would be premature.  But I will still cancel when it calls that method and gets a ``false`` after initial data load.